lib/neo4j-core/query_clauses.rb

```diff
@@ -398,22 +402,23 @@ def from_key_and_value(key, value)
 
         class << self
           def clause_string(clauses)
-            clauses.map!(&:value).join(', ')
+            clauses.map!(&:value).join(Clause::COMMA_SPACE)
           end
         end
       end
 
       class LimitClause < Clause
         @keyword = 'LIMIT'
+        @keyword_downcase = keyword.downcase
```

Comment:
I know, this is just a first pass.

On Sunday, April 5, 2015, Brian Underwood <notifications@github.com> wrote:

> In lib/neo4j-core/query_clauses.rb
> <https://github.com/neo4jrb/neo4j-core/pull/189#discussion_r27781255>:
>
> >            end
> >          end
> >        end
> >
> >        class LimitClause < Clause
> >          @keyword = 'LIMIT'
> > +        @keyword_downcase = keyword.downcase
>
> You might want to consider using a method to get keyword_downcase and
> memoizing it to DRY this up a bit
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neo4jrb/neo4j-core/pull/189/files#r27781255>.
>